napari/utils/_base.py

```diff
@@ -6,9 +6,21 @@
 """
 
 from appdirs import user_config_dir
+from packaging import version
+
+from napari._version import __version__
+
+_dev_version_config_dir = (
+    'dev' if version.parse(__version__).is_devrelease else None
+)
+
 
 _FILENAME = "settings.yaml"
 _APPNAME = "Napari"
 _APPAUTHOR = "Napari"
 _DEFAULT_LOCALE = "en"
-_DEFAULT_CONFIG_PATH = user_config_dir(_APPNAME, _APPAUTHOR, _FILENAME)
+_DEFAULT_CONFIG_PATH = (
+    user_config_dir(_APPNAME, _APPAUTHOR, _dev_version_config_dir)
+    + '/'
```

Comment:
is this cross platform?

Reply:
Hmm. `appdirs.user_config_dir()` returns a string.
_DEFAULT_CONFIG_PATH gets ultimately converted to a Path here for settings:
https://github.com/napari/napari/blob/35390f79f388fd00d3c358ff9d348926d9494b5a/napari/settings/_napari_settings.py#L67
and here for translations:
https://github.com/napari/napari/blob/35390f79f388fd00d3c358ff9d348926d9494b5a/napari/utils/translations.py#L676
(these are the only two places that this var is used.)
So I think we're safe to just do string handling here? Although I think Path() can take a Path and not complain...

Reply:
I am talking about mixing a hardcoded '/', windows uses '\' as separator.

Reply:
Ok, yea I think you're right. It could cause an issue? though it's not popping on tests, maybe the downstream `Path()` when the path is actually used are covering?

Reply:
> Ok, yea I think you're right. It could cause an issue? though it's not popping on tests, maybe the downstream `Path()` when the path is actually used are covering?

in general latest python is ready for this, but you should be replaced with `os.path.join` or use `os.sep` instead of `/`.

Reply:
Actually, I bet this is why `appdirs.user_config_dir` was being abused by passing `settings.yaml` instead of a version (which would make a dir). probably to get around path issues, because  `appdirs.user_config_dir`  returns a system-formatted string.